.github/workflows/romcal-ci.yml

```diff
@@ -23,10 +23,20 @@ jobs:
           node-version-file: '.nvmrc'
 
       # using -q to silent info outputs
-      - run: npm -q --no-progress ci --no-audit
+      - name: Install dependencies
```

Comment:
Inconsistent capitalisation of job names: either use _Title Case_ or _Sentence case_ everywhere.

Reply:
definitely a nitpick, but i'm open to it :)